src/manual/JoystickManager.cpp

```diff
@@ -122,6 +122,10 @@ void JoystickManager::tickJoystickHandlers() {
         joystickHandler.second->tick();
         auto const &[_, world] = world::World::instance();
         auto robot = world->getWorld()->getRobotForId(joystickHandler.second->getCommand().id);
+        if (joystickHandler.second->getCommand().kickSpeed > 0){
+            RTT_DEBUG("KickSpeed = ", joystickHandler.second->getCommand().kickSpeed);
+            RTT_DEBUG("KickType = ", joystickHandler.second->getCommand().kickType == KickType::KICK ? "Kick" : (joystickHandler.second->getCommand().kickType == KickType::CHIP) ? "Chip" : "NoKick");
+        }
```

Comment:
The defining factor whether a robot kicks is the kickType. If the kick type is `NO_KICK`, even if the kickSpeed is above 0, it will not kick. In this case, it might be better to first test if the kickType is not `NO_KICK`, and then print the kickSpeed and the type as either `KICK` or `CHIP`